helm/dashboards/charts/public_dashboards/dashboards/shared/public/policy-status.json

```diff
@@ -0,0 +1,387 @@
+                "color": "green",
+                "value": 90
+              }
+            ]
+          },
+          "unit": "percent"
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 27,
+        "w": 24,
+        "x": 0,
+        "y": 18
+      },
+      "id": 3,
+      "options": {
+        "minVizHeight": 200,
+        "minVizWidth": 200,
+        "orientation": "auto",
+        "reduceOptions": {
+          "calcs": [
+            "lastNotNull"
+          ],
+          "fields": "",
+          "values": false
+        },
+        "showThresholdLabels": true,
+        "showThresholdMarkers": true,
+        "sizing": "auto"
+      },
+      "pluginVersion": "10.2.3",
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "$datasource"
+          },
+          "editorMode": "code",
+          "expr": "(sum(dipstick_policyreport_policy_summary{result=~\"pass|skip\"}) by (cluster_id) / sum(dipstick_policyreport_policy_summary{}) by (cluster_id) * 100)",
+          "hide": false,
+          "instant": false,
+          "legendFormat": "__auto",
+          "range": true,
+          "refId": "B"
+        }
+      ],
+      "title": "Compliance by Cluster",
+      "type": "gauge"
+    }
+  ],
+  "refresh": "",
+  "schemaVersion": 39,
+  "tags": [
+    "owner:team-shield",
+    "topic:workload-cluster",
+    "topic:security",
+    "topic:policy"
+  ],
+  "templating": {
+    "list": [
+      {
+        "allValue": ".*",
+        "current": {
+          "selected": true,
+          "text": [
+            "All"
+          ],
+          "value": [
+            "$__all"
+          ]
+        },
+        "datasource": {
+          "type": "prometheus",
+          "uid": "$datasource"
+        },
+        "definition": "label_values(dipstick_policyreport_policy_summary,cluster_id)",
+        "hide": 0,
+        "includeAll": true,
+        "label": "Cluster",
+        "multi": true,
+        "name": "cluster_id",
+        "options": [],
+        "query": {
+          "qryType": 1,
+          "query": "label_values(dipstick_policyreport_policy_summary,cluster_id)",
+          "refId": "PrometheusVariableQueryEditor-VariableQuery"
+        },
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "type": "query"
+      },
+      {
+        "current": {
+          "selected": false,
+          "text": "default",
+          "value": "default"
+        },
+        "hide": 0,
+        "includeAll": false,
+        "multi": false,
+        "name": "datasource",
+        "options": [],
+        "query": "prometheus",
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "type": "datasource"
+      }
+    ]
+  },
+  "time": {
+    "from": "now-30d",
+    "to": "now"
+  },
+  "timepicker": {},
+  "timezone": "",
+  "title": "Policy Enforcement (PSS) Status",
+  "uid": "policy-status",
```

Comment:
Can you make the uid something more clear ? policy status is not that speaking to me

Reply:
Sure, open to suggestions, but aren't the UIDs usually random strings anyway?

Reply:
Well, we would like it to stop as those are used in URLs but I'm not sure how I would call that one :D

Reply:
Would pod-security-policy-enforcement-overview work ? It's just 40 chars which is the limit ? O therwise it could be pss-enforcement-overview ?

Reply:
We (GS and Shield) have other types of policies which aren't security or PSS which would also be shown here. Policy API is a general purpose policy enforcer :smile:

Reply:
Hmm, what do you think about policy-api-enforcement--overview or similar then?